static/js/ContentText.jsx

```diff
@@ -53,11 +54,12 @@ const VersionImageSpan = ({lang, content, languageToFilter, imageLoadCallback})
     return(<span className={`contentSpan ${lang}`} lang={lang} key={lang}>{content}</span>)
 };
 
-const _filterContentTextByLang = ({text, html, overrideLanguage, defaultToInterfaceOnBilingual=false, bilingualOrder = null}) => {
+const _filterContentTextByLang = ({text, html, markdown, overrideLanguage, defaultToInterfaceOnBilingual=false, bilingualOrder = null}) => {
   /**
    See ContentText for documentation
    */
-  const contentVariable  = html ? html : text;
+  const contentVariable = html ?
+                          html : markdown ? markdown : text;  // assumption is `markdown` or `html` are preferred over `text` if they are present
```

Comment:
can this be:
`const contentVariable = html || markdown || text;`

seems cleaner and achieves the same outcome